### DISCUSSION TITLE
Notification Recipients Device ID's don't clear up even when user credentials change

### DISCUSSION THREAD

**Discussion by ghost** [tutao/tutanota#6354] [Bugs]
Tuta creates a new Notification Recipient Device ID's for each login and doesn't clear them up automatically when user credentials change. The description of this feature is strange as it says that the list of Notification Recipient Device ID's is the list of devices that receive notifications for user. Does that mean that other devices previously logged in continue to receive notifications, even when user credentials change? I think that list should be automatically wiped each time user credentials change...

**Comment by charlag** [comment]
Already implemented: https://github.com/tutao/tutanota/pull/6218
